### PR TITLE
[add] force push

### PR DESCRIPTION
**Pull request by sassbalint**
### Why:

I think the _force push_ step is missing at the end of __Using the BFG__ section.

### What's being changed:

One line added: do the _force push._

### Check off the following:
- [ ] All of the tests are passing.
- [ ] I have reviewed my changes in staging.
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
Thanks so much for opening a PR @sassbalint! I'll get this triaged for review ⚡

**Comment by sassbalint** [comment]
> Thanks so much for opening a PR @sassbalint! I'll get this triaged for review zap

Sorry, am I expected to contribute something? :)

**Comment by janiceilene** [comment]
@sassbalint Thanks for your patience! Our small team is working through lots of reviews and there's nothing else you need to do right now 💛 We'll review this soon and suggest changes or get it merged!

File: content/github/authenticating-to-github/removing-sensitive-data-from-a-repository.md [hunk 1]
```diff
@@ -40,6 +40,12 @@ To replace all text listed in `passwords.txt` wherever it can be found in your r
 $ bfg --replace-text passwords.txt
 ```
 
+At the end, force push the changes:
```
Comment (lecoursen) [inline content/github/authenticating-to-github/removing-sensitive-data-from-a-repository.md:43]:
```suggestion
After the sentence data is removed, you must force push your changes to {% data variables.product.product_name %}.
```

**Review by lecoursen — COMMENTED** [review]
Thanks @sassbalint! I left a suggested change and will merge this PR as soon as you apply it! 🎉

**Comment by walid161hatam** [comment]
Thanks so much for opening a PR @sassbalint <https://github.com/sassbalint>!
I'll get this triaged for review

**Review by lecoursen — APPROVED** [review]

**Pull request merged**